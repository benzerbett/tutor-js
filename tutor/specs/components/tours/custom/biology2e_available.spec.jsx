import Biology2eAvailable from '../../../../src/components/tours/custom/biology2eldavailable';

jest.mock('react-joyride', () => ({
  Tooltip: (props) => <div className={props.className}>{props.step.text}</div>,
}));

describe(Biology2eAvailable, () => {
  it('matches snapshot', () => {
    expect.snapshot(<Biology2eAvailable step={{ style: {}, ride: {} }} />).toMatchSnapshot();
  });
});
