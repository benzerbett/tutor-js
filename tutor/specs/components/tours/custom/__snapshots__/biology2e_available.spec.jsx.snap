// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Biology2eAvailable matches snapshot 1`] = `
<div
  className="centered-wheel mini-notice biology2e-available"
>
  <div
    className="heading"
  >
    <i
      className="tutor-icon fa fa-bullhorn"
      role="presentation"
      type="bullhorn"
    />
    <div
      className="text"
    >
      OpenStax now uses Biology 2e! 
      <br />
      Courses using Biology 1e can no longer be copied.
    </div>
  </div>
  <div
    className="body"
  >
    Biology 2e has new assessments and improved art and explanations. Download our 
    <a
      className="external-icon"
      href="https://d3bxy9euw4e147.cloudfront.net/oscms-prodcms/media/documents/biology_2e_conversion_guide_-_digital.pdf"
      tabIndex={0}
      target="_blank"
    >
      transition guide
    </a>
     to see the key changes from the first edition of Biology to Biology 2e.
  </div>
  <div
    className="footer"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Create a Biology 2e course
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Close
    </button>
  </div>
</div>
`;
